README.md

```diff
@@ -38,20 +38,27 @@ PRs should be posted _before_ class begins on Tuesday, January 2, 2024.
 ### Interview Practice Questions
 
 1. What are three (3) syntax difference between HTML and JSX?
+    - JSX can be used as reuseble components, while HTML cannot.
+    - JSX can contain functions and expressions in the sytnax, while HTML cannot.
+    - JSX can utilize JavaScript conditonals and loops, while HTML cannot
```

Comment:
Kinda. JSX is associated with React and can be in a JavaScript file but the difference would be more focused around the specific of JSX vs HTML rather than JavaScript vs HTML.